src/typing/typingActions.js

```diff
@@ -30,7 +30,10 @@ const typingStatusExpiryLoop = () => async (dispatch: Dispatch, getState: GetSta
 };
 
 /** Start the typing-status expiry loop, if there isn't one already. */
-export const ensureTypingStatusExpiryLoop = () => async (dispatch: Dispatch, getState: GetState) => {
+export const ensureTypingStatusExpiryLoop = () => async (
```

Comment:
To me, this line-break change suggests that you're also affected by the auto-formatting issue I mentioned [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/SOLVED.3A.20Prettier.20auto-formatting/near/985123), hmm. IIUC this should have been done in the "ensureTypingStatusExpiryLoop" rename commit.

Reply:
Ah indeed, thanks. Pushed a fix.